transcribed/page0010.txt

```diff
@@ -1,56 +1,56 @@
-001             TITLE ITS 1.0 3/19/67                                           PAGE 1
-002             MOBY==1
+001              TITLE ITS 1.0 3/19/67                                                 PAGE 1
+002              MOBY== 1
 003
-004             ;AC DEFS
+004              :AC DEFS
 005
-006             A=1
-007             B=1
-008             C=1
-009             D=1
-010             E=1
-011             TT=1
-012             I=1
-013             Q=1
-014             J=1
-015             R=1
-016             P=1     ;DO NOT CHANGE!
-017             T=1     ;"
-018             U=1     ;"
+006              A=1
+007              B=2
+008              C=3
+009              D=4
+010              E=5
+011              TT=6
+012              I=7
+013              Q=10
+014              J=11
+015              R=12
+016              P=15    ;DO NOT CHANGE!
+017              T=16    ;"
+018              U=17    ;"
 019
 020
-021             IOBIT==1        ;IOBIT IN MAP
+021              IOBIT==1        ;IOBIT IN MAP
 022
 023
-024             TTYIF==1        ;RIGHT HALF BITS IN PIRQC
-025             BCNTRZI==2      ;^Z TYPED
-026             BPIPI==4        ;BAD LOC 42
-027             AROIF==10       ;AROV
-028             BILLOP==40      ;ILLEGAL INSTRUCTION
-029             BIOADC==40000   ;MEMORY PROTECTION VIOLATION
-030             BIOCER==400     ;CHANNEL ERROR
-031             VALRTF==200     ;VALUE RETURN
-032             ILLUAD==1000    ;USER DOES NOT HAVE THAT MUCH CORE
-033             BRKF==2000      ;BREAKPOINT FLAG
-034             BUTFUL==4000    ;UTAPE FULL
-035             BILM==10000     ;NON-EX MEM
-036             ;10000  NON EX MEM
-037             ;200000 PDL OV
-038             ;20000  MPV
-039     001 033 BADBTS=BIOADC+BCNTRZI+BPIPI+BIOCER+BILLOP+BILM+VALRTF+BRKF
+024              TTYIF==1        ;RIGHT HALF BITS IN PIRQC
+025              BCNTRZI==2      ;↑Z TYPED
+026              BPIPI==4        ;BAD LOC 42
+027              AROIF==10       ;AROV
+028              BILLOP==40      ;ILLEGAL INSTRUCTION
+029              BIOADC=20000    ;MEMORY PROTECTION VIOLATION
+030              BIOCER=400      ;CHANNEL ERROR
+031              VALRTF==200     ;VALUE RETURN
+032              ILLUAD==1000    ;USER DOES NOT HAVE THAT MUCH CORE
+033              BRKF==2000      ;BREAKPOINT FLAG
+034              BUTFUL==4000    ;UTAPE FULL
+035              BILM==10000     ;NON-EX MEM
+036              ;10000  NON EX MEM
+037              ;200000 PDL OV
+038              ;20000  MPV
+039      001 033 BADBTS=BIOADC+BCNTRZI+BPIPI+BIOCER+BILLOP+BILM+VALRTF+BRKF
 040
-041             TTYIMI==1       ;IMAGE INPUT    ;BITS IN TTYSTS
-042             TTYDDT==2       ;"DDT" MODE
-043             TTYIMO==4       ;IMAGE OUTPUT
+041              TTYIMI==1       ;IMAGE INPUT    ;BITS IN TTYSTS
+042              TTYDDT==2       ;"DDT" MODE
+043              TTYIMO==4       ;IMAGE OUTPUT
 044
-045                             ;BITS IN RH MSKST
```

Comment:
I believe my original is correct.

Reply:
ehm... yes! yes, it is! what did I transcribe?! :)